modules/services/preferences/src/main/java/au/com/shiftyjelly/pocketcasts/preferences/Settings.kt

```diff
@@ -370,8 +370,8 @@ interface Settings {
     val sendCrashReports: UserSetting<Boolean>
     val linkCrashReportsToUser: UserSetting<Boolean>
 
-    fun setEndOfYearShowBadge2022(value: Boolean)
-    fun getEndOfYearShowBadge2022(): Boolean
+    fun setEndOfYearShowBadge2023(value: Boolean)
+    fun getEndOfYearShowBadge2023(): Boolean
 
     fun setEndOfYearModalHasBeenShown(value: Boolean)
     fun getEndOfYearModalHasBeenShown(): Boolean
```

Comment:
This is a minor thing, but I find it a bit confusing that for the modal we're storing whether it has been shown and for the badge we're storing whether it should be shown. It kind of feels like their opposite each other because if one of those settings returns true that means we show it, but for the other setting returns true that means we don't show it. If possible, I'd lean toward making them both return whether the badge/modal should be shown. Definitely not a blocker at all, and may be just my personal preference, so don't feel like you need to change it. It's very possibly not worth making the change at this point, so I'm more sharing just as food for thought.

Reply:
Yeah, it could be confusing, addressed in https://github.com/Automattic/pocket-casts-android/pull/1503/commits/ec7708f7c4d418575740bf90ad5fdbaaae0b37ab